discord/types/widget.py

```diff
@@ -49,13 +49,13 @@ class WidgetMember(User, total=False):
 class _WidgetOptional(TypedDict, total=False):
     channels: List[WidgetChannel]
     members: List[WidgetMember]
-    presence_count: int
 
 
 class Widget(_WidgetOptional):
     id: Snowflake
     name: str
     instant_invite: str
+    presence_count: int
```

Comment:
The `instant_invite` type needs to be `Optional[str]`.

Reply:
Thanks. Is there a reason why `channels` and `members` are left in the `_WidgetOptional` class? They are not listed as optional fields on the API docs.

Reply:
Those can probably be moved too, assuming the documentation doesn't lie (dangerous assumption but oh well).